### ISSUE TITLE
Doesn't seem to work anymore with Leo redesign

### ISSUE THREAD

**Issue by Tobystereo** [psistorm/alfredapp#22]
Hi, 

Leo has been redesigned, not sure if that is the reason, but the plugin doesn't seem to work for me anymore.

Love the plugin though!

Cheers, 

Tobias

**Comment by tknuth** [comment]
I am witnessing the same issue!

**Comment by roest01** [comment]
yes seems Alfred Leo Workflow is dead and can't be updated because of polite e-mail. :(
Any alternatives ?

Quote from readme:
> Today (2017-03-17 09:52) I received a polite e-mail from the company that runs Leo Dictionary asking me to remove all occurrences of this workflow

**Comment by Tobystereo** [comment]
gotcha, that's unfortunate. Thanks for the update.

**Comment by leyyinad** [comment]
Uh, that's sad. I didn't see that. Thank you for the enlightenment.

**Comment by developdude** [comment]
Line 19 in leo.php seems to be … interesting ;-) https

**Comment by m9dfukc** [comment]
Meanwhile, because this broke recently I made my [own](https://github.com/m9dfukc/deepl-alfred-workflow) translation workflow based on [DeepL](https://www.deepl.com), if anybody is interested. 

Since I got the leo workflow working again – thanks to @developdude – I find myself jumping back and forth between this one and my own ;)

**Comment by hmitsch** [comment]
@m9dfukc the DeepL workflow is awesome. Thank you so much. Already loving it!

**Comment by stefanotto** [comment]
Hm, not sure I have the latest workflow (lurking in my Alfred for years without working), but the changes suggested didn't get it to work again. Has something changed at Leo's lately or is the workflow still working for you guys? Thanks!
